src/renderer/src/views/Home/TabPanel.tsx

```diff
@@ -0,0 +1,25 @@
+import { PropsWithChildren } from 'react'
+
+import styled from '@emotion/styled'
+
+const Container = styled.div<{ hide: boolean }>(
+  ({ hide }) => `
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  visibility: ${hide ? 'hidden' : undefined};
```

Comment:
this will result in `visibility: undefined;` when not hidden which isn't valid CSS. We should either wrap the whole line as a conditional eg. `${hide && 'visibility: hidden'};` or provide a valid css value in the falsy return eg `visibility: ${hide ? 'hidden' : 'visible'};`

Reply:
addressed via https://github.com/digidem/mapeo-desktop-shell/pull/23/commits/4dd3660be16afb5b300d492f4dc3e6f39b7d54fb